### DISCUSSION TITLE
Why mqtt library has itself as a dependency ?

### DISCUSSION THREAD

**Discussion by sanduluca** [mqttjs/MQTT.js#1916] [Q&A] [answered]
Why mqtt library has itself as a dependency ?

**Comment by robertsLando** [comment, marked as answer]
Oh that is an error! Seems I added it by error when fixing browser tests here: https://github.com/mqttjs/MQTT.js/commit/80e29a9dc6bb6ad51a7ac968361a550bc1da68cb#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

I may have installed mqtt on root instead of the browser test folder. Fixing it ASAP and thanks for spotting the issue
